distribution/src/resources/config-tool/deployment-full.toml

```diff
@@ -73,6 +73,9 @@ read_timeout = '' # empty by default
 retry_attempts = '' # empty by default
 replace_escape_characters_at_user_login = true   #inferred
 
+[authorization_manager]
+class = "org.wso2.carbon.user.core.authorization.JDBCAuthorizationManager"
```

Comment:
The package name is wrong. It should be "org.wso2.micro.integrator.security.user.core.authorization"